src/actions/selloutaction.js

```diff
@@ -52,20 +52,20 @@ import {
     }
   }
 
-  export const updateSellOutData = (id, data) => async (dispatch) => {
```

Comment:
this selloutaction file is for reference please don't modify it, it will help if anywhere stuck in integration code. let it be as it is

Reply:
this is the one duplicate,  that's why i have commented it